### DISCUSSION TITLE
sending whatsapp messages from linux

### DISCUSSION THREAD

**Discussion by abuakram** [andyholmes/gnome-shell-extension-gsconnect#1534] [Ideas]
Hi... what a great extension...   
It is possible when getting on linux a whatsapp notification from my android, to reply to this message.
I would love to see the possibility to send whatsapp messages from linux using the software installed in my android device, this way, sharing news, links and ideas can be made much easier.  

Thanks

**Comment by andyholmes** [comment]
Unfortunately there is no support in KDE Connect for WhatsApp. This would have to be implemented in the protocol by the upstream project before it can be implemented in GSConnect. Sorry!
